sos/plugins/zfs.py

```diff
@@ -32,5 +32,11 @@ def setup(self):
             "zpool list",
             "zpool status -x"
         ])
+        zpools = self.call_ext_prog("zpool list -H -o name")
+        zpools_list = zpools['output'].splitlines()
+        for zpool in zpools_list:
+            self.add_cmd_output("zpool get all {}".format(zpool),
```

Comment:
Ack to the rest of the patch, but please don't use new-style string formatting unless it is either really needed, or provides a genuine readability or formatting advantage - in this case it is just adding ~6 characters to the line unnecessarily. As a project `sos` pre-dates the new style and some distributions are still shipping Python 2.6 which requires positional indexes for new-style format strings (we are hoping to drop support for it in the next release, but still awaiting confirmation from some downstream maintainers).

There are a handful of uses in the tree that got past review (38 vs. 1325 old-style) but we'd prefer to make a change like this as a planned conversion across the whole code base rather than as a piecemeal effort.